src/workerd/api/wpt/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("//:build/kj_test.bzl", "kj_test")
+load("//:build/wd_cc_library.bzl", "wd_cc_library")
+load("//:build/wd_test.bzl", "wd_test")
+
+[wd_test(
+    src = f,
+    args = [
+      "--experimental",
+    ],
+    data = [
+      f.removesuffix(".wd-test") + ".js",
+      "@wpt//:url",
+    ],
+) for f in glob(
+    ["**/*.wd-test"],
+)]
```

Comment:
Most likely should list each individually rather than using the glob here so that we can selectively import the appropriate wpt filegroups.